<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuSQL</class>
 <widget class="QWidget" name="StuSQL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StuSQL</string>
  </property>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>219</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>239</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_insertname">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>69</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>159</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_insertage">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>69</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>age：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_insertname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>69</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>name：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_insertid">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>69</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1001</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>49</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_insertid">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>69</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>id：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_newid">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>289</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1001</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>179</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_deleteid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>179</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>id：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tablename">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>9</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_updateid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>239</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>id：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_newname">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>289</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_updateid">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>239</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1001</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>119</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tablename">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>9</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>表名：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_insert">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>119</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>9</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_newage">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>289</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>age</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_deleteid">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>179</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1001</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_newage">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>289</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newname">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>289</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_insertage">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>69</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_newid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>289</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>492</width>
     <height>231</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>160</number>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
